internal/guest/runtime/hcsv2/uvm.go

```diff
@@ -363,7 +363,7 @@ func (h *Host) CreateContainer(ctx context.Context, id string, settings *prot.VM
 			if !ok || sid == "" {
 				return nil, errors.Errorf("unsupported 'io.kubernetes.cri.sandbox-id': '%s'", sid)
 			}
-			if err := setupWorkloadContainerSpec(ctx, sid, id, settings.OCISpecification); err != nil {
+			if err := setupWorkloadContainerSpec(ctx, sid, id, settings.OCISpecification, settings.OCIBundlePath); err != nil {
```

Comment:
is bundle directory gets created on tmpfs? Do we expect these logs to grow large?

Reply:
A single nvidia container tool log doesn't get very large. It's only written to when the hook runs, so it's usually a pretty small file.